app/controllers/application_controller.rb

```diff
@@ -22,7 +22,7 @@ def get_favicon
   end
 
   def platform_admin?
-    current_user && current_user.platform_admin
+    current_user.platform_admin
```

Comment:
We might actually need the `current_user &&`. Unless we're certain this will never be called without a user having already logged in.